### PR TITLE
Tutorial how to run something 24/7 and start it on boot

### PR DESCRIPTION
**Pull request by 2020Sanoj**
This PR adds a tutorial how to run something 24/7 and start it on boot

I have read and understood the Contributor's Certificate of Origin at the end of the template, and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: Jonas Löchner | admin@san0j.de

### CODE REVIEW

File: community-tutorials/run-24-7-start-on-boot/01-en.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+reboot
+```
+And after it is rebooted, test if you can see the console (Replace `discord-py-bot` with the name you set in Step 1.1)
+```sh
+screen -r discord-py-bot
+```
+If your want to go back to your normal console simply press `ctrl + a` and then `ctrl + d`
+
+# Conclusion
+Now you have set up a FTP Server. <br>
```
Comment (raphi-0901) [inline community-tutorials/run-24-7-start-on-boot/01-en.md:70]: Wrong conclusion.

File: community-tutorials/run-24-7-start-on-boot/01-en.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+## Step 1.1 - Starting
+In may example i have a python script in /home/python, which is named run.py and i want to start it on boot and it should run 24/7 <br>
+First I go in the folder and install my python requirements that i need for the script
+```sh
+cd /home/python
+pip3 install py-cord py-cord[voice]
+```
+Then I am going to start it (Replace `discord-py-bot` with a name and `python3 run.py` with the command to start)
+```sh
+screen -AmdS discord-py-bot python3 run.py
```
Comment (raphi-0901) [inline community-tutorials/run-24-7-start-on-boot/01-en.md:43]: Could you please amend why we are using screen?

**Review by raphi-0901 — CHANGES_REQUESTED** [review]

File: community-tutorials/run-24-7-start-on-boot/01-en.md (hunk 2 above)
Comment (2020Sanoj) [reply community-tutorials/run-24-7-start-on-boot/01-en.md:43]: done

File: community-tutorials/run-24-7-start-on-boot/01-en.md (hunk 1 above)
Comment (2020Sanoj) [reply community-tutorials/run-24-7-start-on-boot/01-en.md]: is now fixed

**Comment by netcupClaudiaM** [comment]
Hi 2020Sanoj, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Pull request merged**